datapipelines/src/main/kotlin/io/customer/sdk/DataPipelineInstance.kt

```diff
@@ -66,7 +67,7 @@ abstract class DataPipelineInstance : CustomerIOInstance {
      */
     fun identify(userId: String, traits: CustomAttributes) {
         // Method needed for Java interop as inline doesn't work with Java
-        identify(userId = userId, traits = traits, serializationStrategy = JsonAnySerializer.serializersModule.serializer())
+        identify(userId = userId, traits = traits.sanitizeNullsForJson(), serializationStrategy = JsonAnySerializer.serializersModule.serializer())
```

Comment:
The map eventually encodes to JSON using [`toJsonElement`](https://github.com/customerio/cdp-analytics-kotlin/blob/af97bf3e78519da7d9446eb3e9d507c84d3f94e5/core/src/main/java/com/segment/analytics/kotlin/core/utilities/JSON.kt#L396C9-L396C22) which is an extension on `Any` and not `Any?` resulting in crash. The sanitization here maps `null` items to `JsonNull` and prevent the crash. I used the extension only with `CustomAttributes` since `JsonObject` handles `null` on its own without any changes.